### PR TITLE
New show menu and back option

### PR DESCRIPTION
**Pull request by PaudelaGC**

### CODE REVIEW

File: app/helpers/inquirer.js [hunk 1]
```diff
@@ -56,6 +57,30 @@ async function listMainMenu(dbType) {
     }
 }
 
+async function listShowMenu() {
+    let exit = false;
+    while (!exit) {
+        const secondAnswer = await inquirer.prompt([showMenu]);
```
Comment (razondpro) [inline app/helpers/inquirer.js:63]: change secondAnswer variable name to menuOption

File: app/helpers/inquirer.js [hunk 2]
```diff
@@ -56,6 +57,30 @@ async function listMainMenu(dbType) {
...
+                break;
+            case 'Started':
+                //To-do
+                break;
+            case 'Finished':
+                //To-do
+                break;
+            case 'Back':
+                exit = true;
+                console.log("Back to the menu (which was already up)");
```
Comment (razondpro) [inline app/helpers/inquirer.js:76]: delete console logs..

File: app/helpers/questions.js [hunk 3]
```diff
@@ -1,56 +1,70 @@
...
+type: 'input',
+name: 'title',
+message: 'Please write a title for your new task',
 }
 
 const askDetails = {
-  type: 'input',
-  name: 'details',
-  message: 'Please provide the details of the task',
+type: 'input',
```
Comment (razondpro) [inline app/helpers/questions.js:51]: check indentation

File: app/helpers/questions.js [hunk 4]
```diff
@@ -1,56 +1,70 @@
...
+name: 'details',
+message: 'Please provide the details of the task',
 }
 
 const confirmAction = {
-  type: 'confirm',
-  name: 'confirm',
-  message: 'Are you sure?',
+type: 'confirm',
```
Comment (razondpro) [inline app/helpers/questions.js:57]: check indentation

File: app/helpers/questions.js [hunk 5]
```diff
@@ -1,56 +1,70 @@
...
+    'Show tasks',
+    'Delete tasks',
+    'Exit'
+  ],
+  loop: true
+}
+
+const showMenu = {
+type: 'list',
+name: 'menu',
```
Comment (razondpro) [inline app/helpers/questions.js:33]: check indentation

**Comment by PaudelaGC** [comment]
Closed to modify commented changes

**Pull request closed without merging**